CONTRIBUTING.md

```diff
@@ -1,44 +1,20 @@
 # Contributing
 
-Anyone can contribute to the Microclimate Tools for VSCode project, and we welcome your contributions!
+Anyone can contribute to the Microclimate Tools for Eclipse project, and we welcome your contributions! [Visit us on Slack](https://slack-invite-ibm-cloud-tech.mybluemix.net/) if you have any questions.
 
 ## Raising issues
 
-Please raise any bug reports on the project's (put a link here). Be sure to search the list to see if your issue has already been raised.
+If you have identified a problem or have a feature request, we would love to hear about it.
 
-A good bug report is one that make it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
 
-## Developer Certificate of Origin
-
-If you are contributing changes via a pull request, you must sign-off those changes against the *Developer Certificate of Origin*
-
-```
-Developer's Certificate of Origin 1.1
+Search the [list of issues](https://github.com/microclimate-dev2ops/microclimate-eclipse-tools/issues) to see if your issue has already been raised. Also visit our [troubleshooting page](https://microclimate-dev2ops.github.io/mdt-vsc-troubleshooting). If you do not see your issue in either of these places, go ahead and [open a new one](https://github.com/microclimate-dev2ops/microclimate-eclipse-tools/issues/new).
```

Comment:
This needs to be fixed to point to the correct one: https://microclimate-dev2ops.github.io/mdt-vsc-troubleshooting